src/models/blockchains.model.ts

```diff
@@ -25,6 +25,27 @@ class SyncCheckOptions {
   allowance?: number
 }
 
+@model()
+class BlockchainRedirect {
+  @property({
+    type: 'string',
+    required: true,
+  })
+  alias: string
```

Comment:
named this 'alias' instead of blockchain after I saw these being named differently on two redirects for the same chain